petastorm/reader.py

```diff
@@ -67,7 +67,8 @@ def make_reader(dataset_url,
                 cache_type='null', cache_location=None, cache_size_limit=None,
                 cache_row_size_estimate=None, cache_extra_settings=None,
                 hdfs_driver='libhdfs3',
-                transform_spec=None):
+                transform_spec=None,
+                pyarrow_filters=None):
```

Comment:
Please add to the docstring. Don't know if a good idea, but you may update README.rst as it seems like a pretty useful option.

Reply:
I added to the table comparing make_reader and make_batch_reader.